woocommerce/woocommerce-ios
# Order List: fixed cells that become dark when tapped

Fixes #1397  

The scope of the PR is limited to just making the Order Detail screen usable in Dark mode when building the codebase with the release version of Xcode 11.

## Changes
I introduced a new color inside StyleManager, which uses the D1D1D5 color. By default is the background color used by cells when they are tapped with light mode enabled.

## Testing
- Activate dark mode
- Navigate to Order List
- Tap a cell. Check that everything is visually ok (readable, and without dark background)

## Screenshots

| Before             |  After |
:-------------------------:|:-------------------------:
![67500339-d0286380-f682-11e9-8fec-3466ec8fe8d7](https://user-images.githubusercontent.com/495617/68491108-7ca14280-024a-11ea-9ec9-803b597b9e25.PNG) | ![Simulator Screen Shot - iPhone 11 Pro - 2019-11-08 at 17 08 38](https://user-images.githubusercontent.com/495617/68491122-82972380-024a-11ea-8cf6-c34c6d4bca0c.png)


Update release notes:
